t/t.go

```diff
@@ -800,8 +800,10 @@ func downloadDataFiles() {
 		cmd.Dir = *tmp
 
 		if out, err := cmd.CombinedOutput(); err != nil {
-			fmt.Printf("Error %v", err)
-			fmt.Printf("Output %v", out)
+			fmt.Printf("Error %v /n", err)
+			str := string(out)
```

Comment:
we don't need to do this assignment here, we can directly print it

Reply:
Resolved